doc/devel/dependencies.rst

```diff
@@ -168,56 +172,123 @@ remember to clear your artifacts before re-building::
   git clean -xfd
 
 
+Minimum pip / manylinux support (linux)
+---------------------------------------
+
+Matplotlib publishes `manylinux wheels <https://github.com/pypa/manylinux>`_
+which have a minimum version of pip which will recognize the wheels
+
+- Python 3.8: ``manylinx2010`` / pip >=19.0
+- Python 3.9+: ``manylinx2014`` / pip >=19.3
+
+In all cases the required version of pip is embedded in the CPython source.
+
+
+
 .. _development-dependencies:
 
 Additional dependencies for development
 =======================================
 
+.. _setup-dependencies:
+
+Setup Dependencies
+------------------
+
+- `certifi <https://pypi.org/project/certifi/>`_ (>= 2020.06.20).  Used while
+  downloading the freetype and QHull source during build.  This is not a
+  runtime dependency.
+- `setuptools_scm <https://pypi.org/project/setuptools-scm/>`_ (>= 7).  Used to
+  update the reported ``mpl.__version__`` based on the current git commit.
+  Also a runtime dependency for editable installs.
+- `NumPy <https://numpy.org>`_ (>= 1.19).  Also a runtime dependency.
+
+
+.. _compile-dependencies:
+
+C++ Compiler
+------------
+
+Matplotlib requires a C++ compiler that supports C++11.
+
+- `gcc 4.8.1 <https://gcc.gnu.org/projects/cxx-status.html#cxx11>`_ or higher
+- `clang 3.3 <https://clang.llvm.org/cxx_status.html>`_ or higher
+- `Visual Studio 2015
+  <https://docs.microsoft.com/en-us/cpp/overview/visual-cpp-language-conformance?view=msvc-140>`_
+  (aka VS 14.0) or higher
+
+
 .. _test-dependencies:
 
 Additional dependencies for testing
-===================================
+-----------------------------------
```

Comment:
This was intentionally not a subsection of the dev dependencies since you can run the tests without a dev setup. See note at https://matplotlib.org/stable/devel/testing.html#requirements.

While I’m not convinced that it’s necessary to support testing installed packages, we should keep the testing dependencies separate as long as we do that.

Reply:
hmm, fair enough.  I'm not sure that I completely agree, but do not disagree anywhere near enough to argue about it.